cucumber-rails.gemspec

```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency('rdoc', '>= 3.4')
   s.add_development_dependency('yard', '>= 0.8.7')
 
-  s.required_ruby_version = '>= 2.2.0'
+  s.required_ruby_version = '>= 2.3.0'
```

Comment:
`Cucumber-ruby` supports `>= 2.2`, so we should change this back to keep things standardised.